README.md

```diff
@@ -14,7 +14,42 @@ This is a monorepo containing the webapps and libraries in Polymath. In each pac
 # Setup
 
 In the root of the repository run:
-`yarn install`
+
+```
+yarn install
+```
+
+## Installing MongoDB
+
+You will need to run MongoDB in your machine to run the dApps locally
+
+### Windows
+
+Follow [this guide](https://docs.mongodb.com/manual/tutorial/install-mongodb-on-windows/) to install mongodb on windows
+
+### MacOS
+
+You can install mongodb with [Homebrew](https://brew.sh/)
+
+```
+brew update
+brew install mongodb
+```
+
+Mongodb uses `/data/db` as its data folder. If you don't have it you must create
+it
+
+```
+mkdir -p /data/db
```

Comment:
I had to `sudo` for this command to run.

Reply:
Gotcha, adding this